packages/components/psammead-embed-error/README.md

```diff
@@ -16,37 +16,49 @@ npm install @bbc/psammead-embed-error --save
 
 ## Props
 
-| Argument       | Type    | Required | Default | Example                        |
-| -------------- | ------- | -------- | ------- | ------------------------------ |
-| `service`      | String  | No       | `news`  | `arabic`                       |
-| `fillViewport` | Boolean | No       | `false` | `true`                         |
-| `message`      | String  | Yes      | -       | `Sorry, the server's on fire.` |
+| Argument       | Type    | Required | Default | Example                                 |
+| -------------- | ------- | -------- | ------- | --------------------------------------- |
+| `service`      | String  | No       | `news`  | `arabic`                                |
+| `fillViewport` | Boolean | No       | `false` | `true`                                  |
+| `message`      | String  | Yes      | -       | `Sorry, we can't display this content.` |
+| `link`         | Object  | No       | -       | See: [link](#link).                     |
```

Comment:
Should this match `EmbedError.defaultProps` ?
```suggestion
| `link`         | Object  | No       | `null`       | See: [link](#link).                     |
```

Reply:
Yes, it should. Thanks, Olga!